Yoast/javascript
# Add irregular Czech nouns

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements check for irregular Czech nouns (full form exceptions and words with multiple stems); adds a few plural forms to irregular function words.
* [Yoast SEO Free] Adds keyphrase recognition of the most common irregular Czech nouns 

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

Set your website to Czech.
Open a new post and paste a text of min 300 words.
To create a Czech text you can use the [Czech Wikipedia](https://cs.wikipedia.org/wiki/Hlavn%C3%AD_strana).
In Yoast SEO Premium:
- Set `dětmi` as the focus keyphrase
- Add these other forms of the word in the text: `děti, dítě, dítěte, dětem, dítěti, dítětem, dětech` and make sure that all these forms are recognised.
- Set `ucho` as the focus keyphrase
- Add these other forms of the word in the text: `ucho, ucha, uchu, uchem, uši, uším, uší, uších, ušima` and make sure that all these forms are recognised.

Note: It's currently not possible to test these steps in the content-analysis app

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-670
